examples/todo/package.json

```diff
@@ -3,23 +3,28 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "react": "^16.1.1",
-    "react-dom": "^16.1.1",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
     "react-redux": "^5.0.6",
     "redux": "^3.7.2",
-    "resolve-bus-memory": "^0.1.0",
-    "resolve-redux": "^0.1.0",
-    "resolve-scripts": "^0.1.0",
-    "resolve-storage-lite": "^0.1.0"
+    "resolve-bus-memory": "^0.2.1",
+    "resolve-redux": "^0.2.1",
+    "resolve-scripts": "0.2.1",
```

Comment:
An typo or should be fixed version?

Reply:
This is a straight copypasted from the file generated by `create-resolve-app`. If the version is incorrect, we should check the code that generates it

Reply:
Okay, will take it in ming